assets/sass/vendor/_mdc-menu-surface.scss

```diff
@@ -11,5 +11,5 @@
 
 // Outside .googlesitekit-plugin because the open menu is moved to the body.
 .mdc-menu-surface--open {
-	z-index: 11;
```

Comment:
The IB said `greater than 12` although 12 worked for putting the select on top of the Header (as expected. same z-index but appearing later in the DOM means it will be on top)

I don't see any constants though for z-indexes so I'm just following the IB

**before**
![before-select](https://user-images.githubusercontent.com/8496063/115022726-12201780-9ebe-11eb-80b6-73d01e438a3a.png)